src/api/goods.ts

```diff
@@ -6,6 +6,14 @@ export function getAll(): Promise<Good[]> {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => {
+  return getAll()
+    .then(goods => goods.sort((a, b) => a.name.localeCompare(b.name))
+      .filter(good => goods.indexOf(good) < 5));
```

Comment:
```suggestion
  return getAll()
    .then(goods => goods
      .sort((a, b) => a.name.localeCompare(b.name))
      .filter(good => goods.indexOf(good) < 5)
    );
```
use slice instead of filter. Also filter get index as a second argument. So you don't need to use indexOf